index.html

```diff
@@ -1,534 +1,1024 @@
+              interesele clienților.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
 
-  <section class="section" id="presa">
-    <div class="section__title section__title--testimonials">
-      <h2 class="section__description">Vezi ce spun și alții</h2>
-      <h3 class="section__header">
-        Apariții presă
-      </h3>
-    </div>
+    <section class="section section--sign-up">
+      <div class="section__title">
+        <h3 class="section__header">
+          Ai nevoie de asistență juridică personalizată?
+        </h3>
+      </div>
+      <button class="btn">Trimite mesaj</button>
+      <a href="tel:++40742891577"><button class="btn">Sună</button></a>
+    </section>
 
-    <div class="slider">
+    <section class="section" id="presa">
+      <div class="section__title section__title--testimonials">
+        <h2 class="section__description">Vezi ce spun și alții</h2>
+        <h3 class="section__header">Apariții presă</h3>
+      </div>
 
-      <button class="slider__btn slider__btn--left">&larr;</button>
-      <button class="slider__btn slider__btn--right">&rarr;</button>
-      <div class="dots"></div>
-    </div>
-  </section>
+      <div class="slider">
+        <div class="slide" style="transform: translateX(-800%)">
+          <div class="testimonial">
+            <div class="testimonial__details">
+              <span class="testimonial__date">04/02/2023</span>
+              <div classname="testimonial__tags">
+                <span class="testimonial__type">Interviu</span>
+              </div>
+            </div>
+            <h5 class="testimonial__header">
+              Interviu cu Sebastian Nagy, tânărul avocat care o apără pe
+              profesoara ironizată de propriii elevi în online
+            </h5>
+            <blockquote class="testimonial__text">
+              La începutul acestui an, un caz a oripilat opinia publică
+              clujeană. Este vorba despre cazul profesoarei din Cluj-Napoca care
+              a fost victima unor ironizări grosiere din partea elevilor.
+              Profesoara a obținut o prima victorie în instanța, iar dacă va
+              obține și victoria definitivă această urmează să primească
+              despăgubiri din partea părinților celor doi elevi implicați.
+            </blockquote>
+            <address class="testimonial__author">
+              <h6 class="testimonial__name"></h6>
+              <p class="testimonial__location">
+                <a
+                  class="location-link"
+                  href="https://curieruldecluj.ro/sebastian-nagy-tanarul-avocat-care-o-apara-pe-profesoara-ironizata-de-propriii-elevi-in-online-faptele-ilicite-comise-de-catre-parati-sunt-evidente"
+                  target="_blank"
+                  >Curierul de Cluj</a
+                >
+              </p>
+            </address>
+          </div>
+        </div>
+        <div class="slide" style="transform: translateX(-700%)">
```

Comment:
Please refactor inline styles to a dedicated CSS class. For this and following classes I would create CSS classes like `t-8`, `t-7`, etc and use them to keep the HTML more clean.

Reply:
I removed the section because we are rendering dynamically from the server